src/components/ui/dialog.tsx

```diff
@@ -5,7 +5,7 @@ import { CloseButton } from './close-button';
 
 interface DialogContentProps extends ChakraDialog.ContentProps {
   portalled?: boolean;
-  portalRef?: React.RefObject<HTMLElement>;
+  portalRef?: React.RefObject<HTMLElement | null>;
```

Comment:
why are we adding null to the type?

Reply:
it was applied by codemod since .current value can be null initially so React.RefObject<HTMLElement | null> is more accurate definition